app/controllers/stories_controller.rb

```diff
@@ -34,7 +34,7 @@ def index
   def show
     @story_show = true
     path = "/#{params[:username].downcase}/#{params[:slug]}"
-    if (@article = Article.includes(user: :profile).find_by(path: path)&.decorate)
+    if (@article = Article.includes(:user).find_by(path: path)&.decorate)
```

Comment:
I guess this was changed because you got a warning from Bullet that this include is no longer needed, correct?

Reply:
To be honest, I'm not quite sure. @djuber helped me with this :) [Here's more context](https://forem-team.slack.com/archives/C7GE84DEJ/p1631895405212200?thread_ts=1631876143.205700&cid=C7GE84DEJ).